demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -155,7 +155,7 @@ def is_bump_required(self):
 
     def only_docs_changed(self):
         changed_files = self.added_files.union(self.modified_files_in_pack)
-        if (len(changed_files) == 1 and 'README' in changed_files.pop()) or \
+        if (len(changed_files) == 1 and 'README' in changed_files) or \
```

Comment:
```suggestion
        if (len(changed_files) == 1 and 'README' in changed_files[0]) or \
```

shouldn't it be like this?

Reply:
No, it is a set and not a list.

Reply:
```
>>> a = set('a')
>>> a
{'a'}
>>> 'a' in a
True

```

Reply:
But i see your point. Sol is to copy IMO.